index.bs

```diff
@@ -239,28 +241,43 @@ following steps:
     2. If |name| exists as a key in the <a>animator name to animator definition map</a>,
         <a>throw</a> a <a>NotSupportedError</a> and abort all these steps.
 
-    3. If the result of <a>IsConstructor</a>(|animatorCtor|) is false, <a>throw</a> a
+    3. If the result of <a>IsConstructor</a>(|animatorCtorValue|) is false, <a>throw</a> a
         <a>TypeError</a> and abort all these steps.
 
-    4. Let |prototype| be the result of <a>Get</a>(|animatorCtor|, "prototype").
+    4. Let |animatorCtor| be the result of <a>converting</a> animatorCtorValue to the
+        <a>VoidFunction</a> <a>callback function</a> type.  Rethrow any exceptions from the
+        conversion and abort all these steps.
```

Comment:
This needs to be reworded slightly, as it currently always aborts. Something like:

"If an exception is thrown, rethrow the exception and abort all these steps."

Reply:
Done.